import_data/tasks/tasks.py

```diff
@@ -22,44 +22,68 @@
 from .format_stdout import format_stdout
 from .osm_update import osm_update
 
+logging.basicConfig(level=logging.INFO)
+
 cc_exec = concurrent.futures.ThreadPoolExecutor()
 
-logging.basicConfig(level=logging.INFO)
+
+def join(*futures):
+    """
+    Wait for all input futures and raise an exception as soon as one of the
+    futures does so.
+    """
+    run = concurrent.futures.wait(futures, return_when=concurrent.futures.FIRST_EXCEPTION)
+
+    # NOTE: If some day we need to extend this function to return the result of
+    #       input futures, we need to be carefull to keep the order since
+    #       `run.done` is a set.
+    for result in run.done:
+        if result.exception() is not None:
+            raise result.exception()
```

Comment:
As discussed, it could be more explicit to use `raise ... from result.exception()` as the `invoke` command will hide any `UnexpectedExit` exception.